### ISSUE TITLE
Signal 11 crash on iOS in detox 19.4.2

### ISSUE THREAD

**Issue by sandeep-bastola** [wix/Detox#3176]
### Description
iOS app crashes on Detox 19.4.2. As soon as test is started, the app crashes, then detox server could not connect with the app. Hence all tests fail. Signal 11 was raised.
The issue is seen more often in detox 19.4.2 compared to previous versions. 

#### Environment

 - Detox: 19.4.2
 - React Native: 0.66.3
 - Node: 14.15.0
 - Device: iPhone 12
 - Xcode: 13.0
 - iOS: 14.5
 - macOS: 11.5.2
 - Test-runner (select one): `jest-circus`

### DETOX logs
<details>
 <summary>Detox logs</summary>
 <code>
detox[78948] DEBUG: [WSS_CONNECTION, #64251] registered a new connection.
detox[78948] TRACE: [WSS_GET_FROM, #64251] {"params":{"sessionId":"92252d9e-5605-5a46-c2a2-c2cd236b0b13","role":"app"},"messageId":0,"type":"login"}
detox[78948] TRACE: [WSS_SEND_TO, #app] {"params":{"testerConnected":true,"appConnected":true},"messageId":0,"type":"loginSuccess"}
detox[78948] TRACE: [SESSION_JOINED] app joined session 92252d9e-5605-5a46-c2a2-c2cd236b0b13
detox[78948] TRACE: [WSS_SEND_TO, #tester] {"type":"appConnected"}
detox[78948] TRACE: [WS_MESSAGE] {"type":"appConnected"}
 
detox[78948] TRACE: [WS_SEND] {"type":"isReady","params":{},"messageId":-1000}
detox[78948] TRACE: [WSS_GET_FROM, #tester] {"type":"isReady","params":{},"messageId":-1000}
detox[78948] TRACE: [WSS_SEND_TO, #app] {"type":"isReady","params":{},"messageId":-1000}
detox[78948] TRACE: [WSS_GET_FROM, #app] {"messageId":-1000,"params":{},"type":"ready"}
detox[78948] TRACE: [WSS_SEND_TO, #tester] {"messageId":-1000,"params":{},"type":"ready"}
detox[78948] TRACE: [WS_MESSAGE] {"messageId":-1000,"params":{},"type":"ready"}
 
detox[78948] TRACE: [WS_SEND] {"type":"waitForActive","params":{},"messageId":1}
detox[78948] TRACE: [WSS_GET_FROM, #tester] {"type":"waitForActive","params":{},"messageId":1}
detox[78948] TRACE: [WSS_SEND_TO, #app] {"type":"waitForActive","params":{},"messageId":1}
detox[78948] TRACE: [WSS_GET_FROM, #app] {"messageId":-1000,"type":"ready","params":{}}
detox[78948] TRACE: [WSS_SEND_TO, #tester] {"messageId":-1000,"type":"ready","params":{}}
detox[78948] TRACE: [WS_MESSAGE] {"messageId":-1000,"type":"ready","params":{}}
 
detox[78948] TRACE: [WSS_GET_FROM, #app] {"messageId":-1000,"type":"ready","params":{}}
detox[78948] TRACE: [WSS_SEND_TO, #tester] {"messageId":-1000,"type":"ready","params":{}}
detox[78948] TRACE: [WS_MESSAGE] {"messageId":-1000,"type":"ready","params":{}}
 
detox[78948] TRACE: [WSS_GET_FROM, #app] {"params":{},"type":"waitForActiveDone","messageId":1}
detox[78948] TRACE: [WSS_SEND_TO, #tester] {"params":{},"type":"waitForActiveDone","messageId":1}
detox[78948] TRACE: [WS_MESSAGE] {"params":{},"type":"waitForActiveDone","messageId":1}
 
detox[79073] TRACE: [ARTIFACTS_LIFECYCLE] artifactsManager.onAppReady({
  deviceId: '231BD033-82F0-4945-AABC-A1C74857A7A2',
  bundleId: 'world.smokeless.ios.stage',
  pid: 79073
})
detox[78948] TRACE: [ARTIFACTS_LIFECYCLE] artifactsManager.onBeforeTerminateApp({
  deviceId: '231BD033-82F0-4945-AABC-A1C74857A7A2',
  bundleId: 'world.smokeless.ios.stage'
})
detox[78948] DEBUG: [EXEC_CMD, #9] /usr/bin/xcrun simctl terminate 231BD033-82F0-4945-AABC-A1C74857A7A2 world.smokeless.ios.stage
detox[78948] DEBUG: [EXEC_TRY, #9] Terminating world.smokeless.ios.stage...
detox[78948] TRACE: [EXEC_SUCCESS, #9] 
detox[78948] DEBUG: [EXEC_SUCCESS, #9] world.smokeless.ios.stage terminated
detox[78948] TRACE: [ARTIFACTS_LIFECYCLE] artifactsManager.onTerminateApp({
  deviceId: '231BD033-82F0-4945-AABC-A1C74857A7A2',
  bundleId: 'world.smokeless.ios.stage'
})
detox[78948] TRACE: [ARTIFACTS_LIFECYCLE] artifactsManager.onBeforeLaunchApp({
  bundleId: 'world.smokeless.ios.stage',
  deviceId: '231BD033-82F0-4945-AABC-A1C74857A7A2',
  launchArgs: {
    detoxServer: 'ws://localhost:8099',
    detoxSessionId: '92252d9e-5605-5a46-c2a2-c2cd236b0b13'
  }
})
detox[78948] DEBUG: [EXEC_CMD, #10] SIMCTL_CHILD_GULGeneratedClassDisposeDisabled=YES SIMCTL_CHILD_DYLD_INSERT_LIBRARIES="/Users/smokeless/Library/Detox/ios/d71d6b36def467f0017c3f8b6b4189263b4deea3/Detox.framework/Detox" /usr/bin/xcrun simctl launch 231BD033-82F0-4945-AABC-A1C74857A7A2 world.smokeless.ios.stage --args -detoxServer ws://localhost:8099 -detoxSessionId 92252d9e-5605-5a46-c2a2-c2cd236b0b13 -detoxDebugVisibility YES -detoxDisableHierarchyDump YES
detox[78948] DEBUG: [EXEC_TRY, #10] Launching world.smokeless.ios.stage...
detox[78948] TRACE: [SESSION_TORN] app exited session 92252d9e-5605-5a46-c2a2-c2cd236b0b13
detox[78948] TRACE: [WSS_SEND_TO, #tester] {"type":"appDisconnected"}
detox[78948] TRACE: [WS_MESSAGE] {"type":"appDisconnected"}
 
detox[78948] TRACE: [EXEC_SUCCESS, #10] world.smokeless.ios.stage: 79095

detox[78948] DEBUG: [EXEC_CMD, #11] /usr/bin/xcrun simctl get_app_container 231BD033-82F0-4945-AABC-A1C74857A7A2 world.smokeless.ios.stage
detox[78948] TRACE: [EXEC_SUCCESS, #11] /Users/smokeless/Library/Developer/CoreSimulator/Devices/231BD033-82F0-4945-AABC-A1C74857A7A2/data/Containers/Bundle/Application/480C13BE-5CA4-4B93-A6CD-3923CF6F69B4/Smokeless Stage.app

detox[78948] INFO:  [AppleSimUtils.js] world.smokeless.ios.stage launched. To watch simulator logs, run:
        /usr/bin/xcrun simctl spawn 231BD033-82F0-4945-AABC-A1C74857A7A2 log stream --level debug --style compact --predicate 'process == "Smokeless Stage"'
detox[79095] TRACE: [ARTIFACTS_LIFECYCLE] artifactsManager.onLaunchApp({
  bundleId: 'world.smokeless.ios.stage',
  deviceId: '231BD033-82F0-4945-AABC-A1C74857A7A2',
  launchArgs: {
    detoxServer: 'ws://localhost:8099',
    detoxSessionId: '92252d9e-5605-5a46-c2a2-c2cd236b0b13',
    detoxDebugVisibility: 'YES',
    detoxDisableHierarchyDump: 'YES'
  },
  pid: 79095
})
detox[78948] DEBUG: [WSS_CONNECTION, #64283] registered a new connection.
detox[78948] TRACE: [WSS_GET_FROM, #64283] {"messageId":0,"type":"login","params":{"sessionId":"92252d9e-5605-5a46-c2a2-c2cd236b0b13","role":"app"}}
detox[78948] TRACE: [WSS_SEND_TO, #app] {"messageId":0,"type":"loginSuccess","params":{"testerConnected":true,"appConnected":true}}
detox[78948] TRACE: [SESSION_JOINED] app joined session 92252d9e-5605-5a46-c2a2-c2cd236b0b13
detox[78948] TRACE: [WSS_SEND_TO, #tester] {"type":"appConnected"}
detox[78948] TRACE: [WS_MESSAGE] {"type":"appConnected"}
 
detox[78948] TRACE: [WS_SEND] {"type":"isReady","params":{},"messageId":-1000}
detox[78948] TRACE: [WSS_GET_FROM, #tester] {"type":"isReady","params":{},"messageId":-1000}
detox[78948] TRACE: [WSS_SEND_TO, #app] {"type":"isReady","params":{},"messageId":-1000}
detox[78948] TRACE: [WSS_GET_FROM, #app] {"params":{"threadNumber":6,"errorDetails":"Signal 11 was raised\n(\n\t0   Detox                               0x000000010b8966a5 +[NSThread(DetoxUtils) dtx_demangledCallStackSymbols] + 37\n\t1   Detox                               0x000000010b899230 __DTXHandleCrash + 464\n\t2   Detox                               0x000000010b899971 __DTXHandleSignal + 59\n\t3   libsystem_platform.dylib            0x00007fff60335d7d _sigtramp + 29\n\t4   ???                                 0x0000000000000000 0x0 + 0\n\t5   CoreFoundation                      0x00000001105c7833 -[__NSDictionaryM dealloc] + 128\n\t6   libobjc.A.dylib                     0x000000011011b604 objc_object::sidetable_release(bool, bool) + 174\n\t7   libobjc.A.dylib                     0x00000001101178ad _object_remove_assocations + 562\n\t8   libobjc.A.dylib                     0x0000000110114c02 objc_destructInstance + 84\n\t9   libobjc.A.dylib                     0x000000011011ad88 -[NSObject dealloc] + 21\n\t10  FBSDKCoreKit                        0x000000010cf75a40 -[FBSDKGraphRequestConnection dealloc] + 96\n\t11  libobjc.A.dylib                     0x000000011011b604 objc_object::sidetable_release(bool, bool) + 174\n\t12  CFNetwork                           0x000000010c0ad610 _CFNetworkHTTPConnectionCacheSetLimit + 162696\n\t13  CFNetwork                           0x000000010bef6183 CFNetwork + 24963\n\t14  DetoxSync                           0x0000000143f65b27 ____detox_sync_dispatch_wrapper_block_invoke + 23\n\t15  libdispatch.dylib                   0x000000011489670d _dispatch_call_block_and_release + 12\n\t16  libdispatch.dylib                   0x00000001148978df _dispatch_client_callout + 8\n\t17  libdispatch.dylib                   0x000000011489de15 _dispatch_lane_serial_drain + 715\n\t18  libdispatch.dylib                   0x000000011489e9c3 _dispatch_lane_invoke + 455\n\t19  libdispatch.dylib                   0x00000001148a8f81 _dispatch_workloop_worker_thread + 772\n\t20  libsystem_pthread.dylib             0x00007fff6034045d _pthread_wqthread + 314\n\t21  libsystem_pthread.dylib             0x00007fff6033f42f start_wqthread + 15\n)","queueName":"com.apple.NSURLSession-work"},"type":"AppWillTerminateWithError","messageId":-10000}
detox[78948] TRACE: [WSS_SEND_TO, #tester] {"params":{"threadNumber":6,"errorDetails":"Signal 11 was raised\n(\n\t0   Detox                               0x000000010b8966a5 +[NSThread(DetoxUtils) dtx_demangledCallStackSymbols] + 37\n\t1   Detox                               0x000000010b899230 __DTXHandleCrash + 464\n\t2   Detox                               0x000000010b899971 __DTXHandleSignal + 59\n\t3   libsystem_platform.dylib            0x00007fff60335d7d _sigtramp + 29\n\t4   ???                                 0x0000000000000000 0x0 + 0\n\t5   CoreFoundation                      0x00000001105c7833 -[__NSDictionaryM dealloc] + 128\n\t6   libobjc.A.dylib                     0x000000011011b604 objc_object::sidetable_release(bool, bool) + 174\n\t7   libobjc.A.dylib                     0x00000001101178ad _object_remove_assocations + 562\n\t8   libobjc.A.dylib                     0x0000000110114c02 objc_destructInstance + 84\n\t9   libobjc.A.dylib                     0x000000011011ad88 -[NSObject dealloc] + 21\n\t10  FBSDKCoreKit                        0x000000010cf75a40 -[FBSDKGraphRequestConnection dealloc] + 96\n\t11  libobjc.A.dylib                     0x000000011011b604 objc_object::sidetable_release(bool, bool) + 174\n\t12  CFNetwork                           0x000000010c0ad610 _CFNetworkHTTPConnectionCacheSetLimit + 162696\n\t13  CFNetwork                           0x000000010bef6183 CFNetwork + 24963\n\t14  DetoxSync                           0x0000000143f65b27 ____detox_sync_dispatch_wrapper_block_invoke + 23\n\t15  libdispatch.dylib                   0x000000011489670d _dispatch_call_block_and_release + 12\n\t16  libdispatch.dylib                   0x00000001148978df _dispatch_client_callout + 8\n\t17  libdispatch.dylib                   0x000000011489de15 _dispatch_lane_serial_drain + 715\n\t18  libdispatch.dylib                   0x000000011489e9c3 _dispatch_lane_invoke + 455\n\t19  libdispatch.dylib                   0x00000001148a8f81 _dispatch_workloop_worker_thread + 772\n\t20  libsystem_pthread.dylib             0x00007fff6034045d _pthread_wqthread + 314\n\t21  libsystem_pthread.dylib             0x00007fff6033f42f start_wqthread + 15\n)","queueName":"com.apple.NSURLSession-work"},"type":"AppWillTerminateWithError","messageId":-10000}
 

</code>
</details>

<details>
 <summary>Device logs</summary>
 <code>
2022-01-18 09:29:44.203 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKFeatureManager.FBSDKFeatureAAM in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.203 Db Smokeless Stage[80550:6ab37f] [com.apple.network:activity] Returning should log: 0 for activity cfnetwork:foreground_task
2022-01-18 09:29:44.203 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKSettingsAdvertisingTrackingStatus in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.203 Df Smokeless Stage[80550:6ab37f] (CFNetwork) [Telemetry]: Activity <nw_activity 12:2 [7D0E1081-D565-4B36-81DA-25EB6B1A4192] (reporting strategy default)> on Task <2F491C89-9CBE-4817-94BD-0E35357230F3>.<1> was not selected for reporting
2022-01-18 09:29:44.203 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKFeatureManager.FBSDKFeaturePrivacyProtection in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.204 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKFeatureManager.FBSDKFeatureSKAdNetwork in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.204 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKFeatureManager.FBSDKFeatureSKAdNetworkConversionValue in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.204 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKFeatureManager.FBSDKFeatureAEM in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.203 Df Smokeless Stage[80550:6ab4c2] (CFNetwork) Connection 5: starting, TC(0x0)
2022-01-18 09:29:44.204 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKFeatureManager.FBSDKFeatureRestrictiveDataFiltering in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.204 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKFeatureManager.FBSDKFeatureEventDeactivation in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.204 Df Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] [C5 BC566B0C-4A57-4356-818C-F3DD4C2FC8DE graph.facebook.com:443 tcp, url hash: 3593ae5e, tls, context: com.apple.CFNetwork.NSURLSession.{61A3D505-96A1-42E6-B397-344F53C03547}{(null)}{Y}{2}, proc: D6C13870-EF95-30D6-82EC-D09C6A8801F9] start
2022-01-18 09:29:44.204 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKFeatureManager.FBSDKFeatureATELogging in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.204 I  Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_endpoint_handler_start [C5 graph.facebook.com:443 initial path ((null))]
2022-01-18 09:29:44.204 Df Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] [C5 graph.facebook.com:443 initial path ((null))] event: path:start @0.000s
2022-01-18 09:29:44.204 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKFeatureManager.FBSDKFeatureCodelessEvents in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.202 Db Smokeless Stage[80550:6ab3c9] [com.apple.defaults:User Defaults] found no value for key UseNewAXBundleLoader in CFPrefsSearchListSource<0x600000cc8e00> (Domain: com.apple.Accessibility, Container: /Users/smokeless/Library/Developer/CoreSimulator/Devices/231BD033-82F0-4945-AABC-A1C74857A7A2/data)
2022-01-18 09:29:44.205 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKFeatureManager.FBSDKFeatureAAM in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.205 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKSettingsAdvertisingTrackingStatus in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.205 Db Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_endpoint_handler_path_change [C5 graph.facebook.com:443 waiting path (satisfied (Path is satisfied), interface: en0)]
2022-01-18 09:29:44.205 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKFeatureManager.FBSDKFeaturePrivacyProtection in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.205 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKFeatureManager.FBSDKFeatureSKAdNetwork in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.205 Df Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] [C5 graph.facebook.com:443 waiting path (satisfied (Path is satisfied), interface: en0)] event: path:satisfied @0.000s, uuid: C789301F-A8E7-4F75-B7A8-FFAF70DCE423
2022-01-18 09:29:44.205 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKFeatureManager.FBSDKFeatureSKAdNetworkConversionValue in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.205 I  Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_connection_endpoint_report_on_nw_queue [C5 graph.facebook.com:443 waiting path (satisfied (Path is satisfied), interface: en0)] skipping state update
2022-01-18 09:29:44.205 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKFeatureManager.FBSDKFeatureAEM in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.205 Db Smokeless Stage[80550:6ab4c2] [com.apple.network:] nw_endpoint_proxy_handler_should_use_proxy Looking up proxy for hostname: graph.facebook.com, ifindex: 0
2022-01-18 09:29:44.206 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKFeatureManager.FBSDKFeatureRestrictiveDataFiltering in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.205 Db Smokeless Stage[80550:6ab3c9] [com.apple.defaults:User Defaults] found no value for key UseNewAXBundleLoader in CFPrefsSearchListSource<0x600000cc0980> (Domain: com.apple.Accessibility, Container: (null))
2022-01-18 09:29:44.206 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKFeatureManager.FBSDKFeatureEventDeactivation in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.206 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKFeatureManager.FBSDKFeatureATELogging in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.206 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKFeatureManager.FBSDKFeatureCodelessEvents in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.206 Db Smokeless Stage[80550:6ab4c2] [com.apple.SystemConfiguration:SCPreferences] create <SCPreferences 0x7fad15d05fd0 [0x10adb5b90]> {name = SCDynamicStoreCopyProxiesWithOptions, id = /Library/Managed Preferences/mobile/com.apple.SystemConfiguration.plist, path = /Library/Managed Preferences/mobile/com.apple.SystemConfiguration.plist}
2022-01-18 09:29:44.206 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKFeatureManager.FBSDKFeatureAAM in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.206 Db Smokeless Stage[80550:6ab4c2] [com.apple.SystemConfiguration:SCPreferences] SCPreferences() access: /Library/Managed Preferences/mobile/com.apple.SystemConfiguration.plist, size=0
2022-01-18 09:29:44.206 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKSettingsAdvertisingTrackingStatus in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.206 Db Smokeless Stage[80550:6ab4c2] [com.apple.SystemConfiguration:SCPreferences] release <SCPreferences 0x7fad15d05fd0 [0x10adb5b90]> {name = SCDynamicStoreCopyProxiesWithOptions, id = /Library/Managed Preferences/mobile/com.apple.SystemConfiguration.plist, path = /Library/Managed Preferences/mobile/com.apple.SystemConfiguration.plist, accessed}
2022-01-18 09:29:44.206 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKFeatureManager.FBSDKFeaturePrivacyProtection in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.206 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKFeatureManager.FBSDKFeatureSKAdNetwork in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.206 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKFeatureManager.FBSDKFeatureSKAdNetworkConversionValue in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.206 Db Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] -[NWConcrete_nw_endpoint_resolver startWithHandler:] [C5 graph.facebook.com:443 waiting resolver (satisfied (Path is satisfied), interface: en0)]
2022-01-18 09:29:44.206 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKFeatureManager.FBSDKFeatureAEM in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.207 Df Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] [C5 graph.facebook.com:443 in_progress resolver (satisfied (Path is satisfied), interface: en0)] event: resolver:start_dns @0.002s
2022-01-18 09:29:44.207 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key com.facebook.sdk:FBSDKSettingsAdvertisingTrackingStatus in CFPrefsSearchListSource<0x600000cc4500> (Domain: world.smokeless.ios.stage, Container: (null))
2022-01-18 09:29:44.207 Df Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_connection_report_state_with_handler_on_nw_queue [C5] reporting state preparing
2022-01-18 09:29:44.207 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] setting {
    "com.facebook.sdk:FBSDKAppEventsConfiguration" = {length = 383, bytes = 0x62706c69 73743030 d4010203 04050607 ... 00000000 00000127 };
} in CFPrefsPlistSource<0x600000cc0280> (Domain: world.smokeless.ios.stage, User: kCFPreferencesCurrentUser, ByHost: No, Container: (null), Contents Need Refresh: No)
2022-01-18 09:29:44.207 I  Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_resolver_create_dns_service_locked [C5] Starting host resolution graph.facebook.com:443, flags 0xc000d000 proto 0
2022-01-18 09:29:44.208 Df Smokeless Stage[80550:6ab4c2] (CFNetwork) Task <65FB8707-3E5D-420E-8A56-AC08472E5579>.<1> setting up Connection 5
2022-01-18 09:29:44.208 Db Smokeless Stage[80550:6ab4c2] [com.apple.network:activity] Returning should log: 0 for activity cfnetwork:foreground_task
2022-01-18 09:29:44.208 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] setting {
    "com.facebook.sdk:FBSDKAppEventsConfigurationTimestamp" = "2022-01-18 08:29:44 +0000";
} in CFPrefsPlistSource<0x600000cc0280> (Domain: world.smokeless.ios.stage, User: kCFPreferencesCurrentUser, ByHost: No, Container: (null), Contents Need Refresh: No)
2022-01-18 09:29:44.208 Df Smokeless Stage[80550:6ab4c2] [com.apple.CFNetwork:ATS] Task <FFF4EA71-83E4-4C40-B62E-F8B8513E19EB>.<1> {strength 1, tls 8, ct 0, sub 0, sig 0, ciphers 1, bundle 0, builtin 0}
2022-01-18 09:29:44.208 Df Smokeless Stage[80550:6ab4c2] [com.apple.CFNetwork:ATS] Task <2F491C89-9CBE-4817-94BD-0E35357230F3>.<1> {strength 1, tls 8, ct 0, sub 0, sig 0, ciphers 1, bundle 0, builtin 0}
2022-01-18 09:29:44.208 Db Smokeless Stage[80550:6ab3c9] [com.apple.defaults:User Defaults] found no value for key UseNewAXBundleLoader in CFPrefsSearchListSource<0x600000cc8e00> (Domain: com.apple.Accessibility, Container: /Users/smokeless/Library/Developer/CoreSimulator/Devices/231BD033-82F0-4945-AABC-A1C74857A7A2/data)
2022-01-18 09:29:44.208 Db Smokeless Stage[80550:6ab3c9] [com.apple.defaults:User Defaults] found no value for key UseNewAXBundleLoader in CFPrefsSearchListSource<0x600000cc0980> (Domain: com.apple.Accessibility, Container: (null))
2022-01-18 09:29:44.208 I  Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_resolver_host_resolve_callback [C5] flags=0x40000003 ifindex=0 error=NoError(0) hostname=star.c10r.facebook.com. addr=2a03:2880:f045:12:face:b00c:0:2 ttl=77
2022-01-18 09:29:44.208 I  Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_resolver_host_resolve_callback [C5] flags=0x40000002 ifindex=0 error=NoError(0) hostname=star.c10r.facebook.com. addr=157.240.201.17 ttl=77
2022-01-18 09:29:44.208 Db Smokeless Stage[80550:6ab4c2] [com.apple.network:] sa_dst_compare_statistics no preference, equal min rtt 11.000000
2022-01-18 09:29:44.208 Db Smokeless Stage[80550:6ab4c2] [com.apple.network:] sa_dst_compare_rfc6724 Rule 6, prefer d2, d2 precedence 40 > d1 precedence 35
2022-01-18 09:29:44.208 Db Smokeless Stage[80550:6ab4c2] [com.apple.network:] sa_dst_compare_internal <private>@0 < <private>@0
2022-01-18 09:29:44.208 Db Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_endpoint_resolver_update [C5 graph.facebook.com:443 in_progress resolver (satisfied (Path is satisfied), interface: en0)] resolver is complete
2022-01-18 09:29:44.209 I  Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_endpoint_resolver_update [C5 graph.facebook.com:443 in_progress resolver (satisfied (Path is satisfied), interface: en0)] Adding endpoint handler for 2a03:2880:f045:12:face:b00c:0:2.443
2022-01-18 09:29:44.209 Db Smokeless Stage[80550:6aaec6] [com.apple.CFBundle:strings] Bundle: CFBundle 0x7fad05d04080 </Users/smokeless/Library/Developer/CoreSimulator/Devices/231BD033-82F0-4945-AABC-A1C74857A7A2/data/Containers/Bundle/Application/1E9F0FAE-9305-40E9-B935-8DF987A5B6C9/Smokeless Stage.app> (executable, loaded), key: ErrorRecovery.OK, value: OK, table: FacebookSDK, localizationName: (null), result: OK
2022-01-18 09:29:44.209 Db Smokeless Stage[80550:6aaec6] [com.apple.CFBundle:strings] Bundle: CFBundle 0x7fad05d04080 </Users/smokeless/Library/Developer/CoreSimulator/Devices/231BD033-82F0-4945-AABC-A1C74857A7A2/data/Containers/Bundle/Application/1E9F0FAE-9305-40E9-B935-8DF987A5B6C9/Smokeless Stage.app> (executable, loaded), key: ErrorRecovery.Cancel, value: Cancel, table: FacebookSDK, localizationName: (null), result: Cancel
2022-01-18 09:29:44.209 I  Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_endpoint_resolver_update [C5 graph.facebook.com:443 in_progress resolver (satisfied (Path is satisfied), interface: en0)] Adding endpoint handler for 157.240.201.17:443
2022-01-18 09:29:44.209 Db Smokeless Stage[80550:6aaec6] [com.apple.CFBundle:strings] Bundle: CFBundle 0x7fad05d04080 </Users/smokeless/Library/Developer/CoreSimulator/Devices/231BD033-82F0-4945-AABC-A1C74857A7A2/data/Containers/Bundle/Application/1E9F0FAE-9305-40E9-B935-8DF987A5B6C9/Smokeless Stage.app> (executable, loaded), key: ErrorRecovery.Transient.Suggestion, value: The server is temporarily busy, please try again., table: FacebookSDK, localizationName: (null), result: The server is temporarily busy, please try again.
2022-01-18 09:29:44.209 Db Smokeless Stage[80550:6aaec6] [com.apple.CFBundle:strings] Bundle: CFBundle 0x7fad05d04080 </Users/smokeless/Library/Developer/CoreSimulator/Devices/231BD033-82F0-4945-AABC-A1C74857A7A2/data/Containers/Bundle/Application/1E9F0FAE-9305-40E9-B935-8DF987A5B6C9/Smokeless Stage.app> (executable, loaded), key: ErrorRecovery.Login.Suggestion, value: Please log into this app again to reconnect your Facebook account., table: FacebookSDK, localizationName: (null), result: Please log into this app again to reconnect your Facebook account.
2022-01-18 09:29:44.209 Db Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_endpoint_resolver_update [C5 graph.facebook.com:443 in_progress resolver (satisfied (Path is satisfied), interface: en0)] Updated endpoint list is (2a03:2880:f045:12:face:b00c:0:2.443,157.240.201.17:443)
2022-01-18 09:29:44.209 Df Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] [C5 graph.facebook.com:443 in_progress resolver (satisfied (Path is satisfied), interface: en0)] event: resolver:receive_dns @0.004s
2022-01-18 09:29:44.209 I  Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_connection_create_with_id [C6] create connection to Hostname#1b540f1a:443
2022-01-18 09:29:44.209 Df Smokeless Stage[80550:6ab4c2] (CFNetwork) Connection 6: enabling TLS
2022-01-18 09:29:44.210 Df Smokeless Stage[80550:6ab4c2] (CFNetwork) Connection 6: starting, TC(0x0)
2022-01-18 09:29:44.210 Df Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] [C6 7A283221-1927-4BE6-B539-CA35F6169087 graph.facebook.com:443 tcp, url hash: aa2709a7, tls, context: com.apple.CFNetwork.NSURLSession.{CFEDE6C6-271A-4184-9B7A-654D677CAB08}{(null)}{Y}{2}, proc: D6C13870-EF95-30D6-82EC-D09C6A8801F9] start
2022-01-18 09:29:44.210 I  Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_endpoint_handler_start [C6 graph.facebook.com:443 initial path ((null))]
2022-01-18 09:29:44.210 Df Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] [C6 graph.facebook.com:443 initial path ((null))] event: path:start @0.000s
2022-01-18 09:29:44.210 Db Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_endpoint_handler_path_change [C6 graph.facebook.com:443 waiting path (satisfied (Path is satisfied), interface: en0)]
2022-01-18 09:29:44.210 Df Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] [C6 graph.facebook.com:443 waiting path (satisfied (Path is satisfied), interface: en0)] event: path:satisfied @0.000s, uuid: 67B47B9C-CCBD-4E78-8C6F-1C5D37759065
2022-01-18 09:29:44.210 I  Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_connection_endpoint_report_on_nw_queue [C6 graph.facebook.com:443 waiting path (satisfied (Path is satisfied), interface: en0)] skipping state update
2022-01-18 09:29:44.210 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] setting {
    "com.facebook.sdk:serverConfiguration252927988890514" = {length = 3339, bytes = 0x62706c69 73743030 d4000100 02000300 ... 00000000 00000a6f };
} in CFPrefsPlistSource<0x600000cc0280> (Domain: world.smokeless.ios.stage, User: kCFPreferencesCurrentUser, ByHost: No, Container: (null), Contents Need Refresh: No)
2022-01-18 09:29:44.210 Db Smokeless Stage[80550:6ab4c2] [com.apple.network:] nw_endpoint_proxy_handler_should_use_proxy Looking up proxy for hostname: graph.facebook.com, ifindex: 0
2022-01-18 09:29:44.211 Db Smokeless Stage[80550:6ab3c9] [com.apple.defaults:User Defaults] found no value for key UseNewAXBundleLoader in CFPrefsSearchListSource<0x600000cc8e00> (Domain: com.apple.Accessibility, Container: /Users/smokeless/Library/Developer/CoreSimulator/Devices/231BD033-82F0-4945-AABC-A1C74857A7A2/data)
2022-01-18 09:29:44.211 Db Smokeless Stage[80550:6ab4c2] [com.apple.SystemConfiguration:SCPreferences] create <SCPreferences 0x7facf5d16250 [0x10adb5b90]> {name = SCDynamicStoreCopyProxiesWithOptions, id = /Library/Managed Preferences/mobile/com.apple.SystemConfiguration.plist, path = /Library/Managed Preferences/mobile/com.apple.SystemConfiguration.plist}
2022-01-18 09:29:44.211 Db Smokeless Stage[80550:6ab3c9] [com.apple.defaults:User Defaults] found no value for key UseNewAXBundleLoader in CFPrefsSearchListSource<0x600000cc0980> (Domain: com.apple.Accessibility, Container: (null))
2022-01-18 09:29:44.211 Db Smokeless Stage[80550:6ab4c2] [com.apple.SystemConfiguration:SCPreferences] SCPreferences() access: /Library/Managed Preferences/mobile/com.apple.SystemConfiguration.plist, size=0
2022-01-18 09:29:44.211 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key UseNewAXBundleLoader in CFPrefsSearchListSource<0x600000cc8e00> (Domain: com.apple.Accessibility, Container: /Users/smokeless/Library/Developer/CoreSimulator/Devices/231BD033-82F0-4945-AABC-A1C74857A7A2/data)
2022-01-18 09:29:44.211 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key UseNewAXBundleLoader in CFPrefsSearchListSource<0x600000cc0980> (Domain: com.apple.Accessibility, Container: (null))
2022-01-18 09:29:44.211 Db Smokeless Stage[80550:6ab4c2] [com.apple.SystemConfiguration:SCPreferences] release <SCPreferences 0x7facf5d16250 [0x10adb5b90]> {name = SCDynamicStoreCopyProxiesWithOptions, id = /Library/Managed Preferences/mobile/com.apple.SystemConfiguration.plist, path = /Library/Managed Preferences/mobile/com.apple.SystemConfiguration.plist, accessed}
2022-01-18 09:29:44.212 Db Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] -[NWConcrete_nw_endpoint_resolver startWithHandler:] [C6 graph.facebook.com:443 waiting resolver (satisfied (Path is satisfied), interface: en0)]
2022-01-18 09:29:44.213 Df Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] [C6 graph.facebook.com:443 in_progress resolver (satisfied (Path is satisfied), interface: en0)] event: resolver:start_dns @0.003s
2022-01-18 09:29:44.213 Df Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_connection_report_state_with_handler_on_nw_queue [C6] reporting state preparing
2022-01-18 09:29:44.214 I  Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_resolver_create_dns_service_locked [C6] Starting host resolution graph.facebook.com:443, flags 0xc000d000 proto 0
2022-01-18 09:29:44.214 Df Smokeless Stage[80550:6ab4c2] (CFNetwork) Task <F43E435F-E74B-4CF0-8BBC-18F373934BBE>.<1> setting up Connection 6
2022-01-18 09:29:44.214 Db Smokeless Stage[80550:6ab4c2] [com.apple.network:activity] Returning should log: 0 for activity cfnetwork:foreground_task
2022-01-18 09:29:44.214 Db Smokeless Stage[80550:6ab4c2] [com.apple.CFNetwork:Coalescing] removing all entries config 0x600002285a60
2022-01-18 09:29:44.214 I  Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_connection_create_with_id [C7] create connection to Hostname#1b540f1a:443
2022-01-18 09:29:44.215 Df Smokeless Stage[80550:6ab4c2] (CFNetwork) Connection 7: enabling TLS
2022-01-18 09:29:44.215 Df Smokeless Stage[80550:6ab4c2] (CFNetwork) Connection 7: starting, TC(0x0)
2022-01-18 09:29:44.215 Df Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] [C7 9846BBC7-059B-4D81-B7C7-8FD5E9B9DE1E graph.facebook.com:443 tcp, url hash: dd6c8d5a, tls, context: com.apple.CFNetwork.NSURLSession.{9090FD05-4D92-4361-BABD-AAAB4F06CEDB}{(null)}{Y}{2}, proc: D6C13870-EF95-30D6-82EC-D09C6A8801F9] start
2022-01-18 09:29:44.215 I  Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_endpoint_handler_start [C7 graph.facebook.com:443 initial path ((null))]
2022-01-18 09:29:44.215 Df Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] [C7 graph.facebook.com:443 initial path ((null))] event: path:start @0.000s
2022-01-18 09:29:44.215 Db Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_endpoint_handler_path_change [C7 graph.facebook.com:443 waiting path (satisfied (Path is satisfied), interface: en0)]
2022-01-18 09:29:44.215 Df Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] [C7 graph.facebook.com:443 waiting path (satisfied (Path is satisfied), interface: en0)] event: path:satisfied @0.000s, uuid: A501716C-F490-4EF9-9403-A2EC73539FD7
2022-01-18 09:29:44.215 I  Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_connection_endpoint_report_on_nw_queue [C7 graph.facebook.com:443 waiting path (satisfied (Path is satisfied), interface: en0)] skipping state update
2022-01-18 09:29:44.216 Db Smokeless Stage[80550:6ab4c2] [com.apple.network:] nw_endpoint_proxy_handler_should_use_proxy Looking up proxy for hostname: graph.facebook.com, ifindex: 0
2022-01-18 09:29:44.216 Db Smokeless Stage[80550:6ab4c2] [com.apple.SystemConfiguration:SCPreferences] create <SCPreferences 0x7fad15c2bb10 [0x10adb5b90]> {name = SCDynamicStoreCopyProxiesWithOptions, id = /Library/Managed Preferences/mobile/com.apple.SystemConfiguration.plist, path = /Library/Managed Preferences/mobile/com.apple.SystemConfiguration.plist}
2022-01-18 09:29:44.216 Db Smokeless Stage[80550:6ab4c2] [com.apple.SystemConfiguration:SCPreferences] SCPreferences() access: /Library/Managed Preferences/mobile/com.apple.SystemConfiguration.plist, size=0
2022-01-18 09:29:44.216 Db Smokeless Stage[80550:6ab4c2] [com.apple.SystemConfiguration:SCPreferences] release <SCPreferences 0x7fad15c2bb10 [0x10adb5b90]> {name = SCDynamicStoreCopyProxiesWithOptions, id = /Library/Managed Preferences/mobile/com.apple.SystemConfiguration.plist, path = /Library/Managed Preferences/mobile/com.apple.SystemConfiguration.plist, accessed}
2022-01-18 09:29:44.216 Db Smokeless Stage[80550:6ab3c9] [com.apple.defaults:User Defaults] found no value for key UseNewAXBundleLoader in CFPrefsSearchListSource<0x600000cc8e00> (Domain: com.apple.Accessibility, Container: /Users/smokeless/Library/Developer/CoreSimulator/Devices/231BD033-82F0-4945-AABC-A1C74857A7A2/data)
2022-01-18 09:29:44.216 Db Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] -[NWConcrete_nw_endpoint_resolver startWithHandler:] [C7 graph.facebook.com:443 waiting resolver (satisfied (Path is satisfied), interface: en0)]
2022-01-18 09:29:44.216 Db Smokeless Stage[80550:6ab3c9] [com.apple.defaults:User Defaults] found no value for key UseNewAXBundleLoader in CFPrefsSearchListSource<0x600000cc0980> (Domain: com.apple.Accessibility, Container: (null))
2022-01-18 09:29:44.217 Df Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] [C7 graph.facebook.com:443 in_progress resolver (satisfied (Path is satisfied), interface: en0)] event: resolver:start_dns @0.001s
2022-01-18 09:29:44.217 Df Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_connection_report_state_with_handler_on_nw_queue [C7] reporting state preparing
2022-01-18 09:29:44.217 E  Smokeless Stage[80550:6ab3ca] [com.wix.Detox:DetoxManager] App crashed: Signal 11 was raised
(
	0   Detox                               0x0000000105e046a5 +[NSThread(DetoxUtils) dtx_demangledCallStackSymbols] + 37
	1   Detox                               0x0000000105e07230 __DTXHandleCrash + 464
	2   Detox                               0x0000000105e07971 __DTXHandleSignal + 59
	3   libsystem_platform.dylib            0x00007fff60335d7d _sigtramp + 29
	4   ???                                 0x0000000000000000 0x0 + 0
	5   CoreFoundation                      0x000000010ab35833 -[__NSDictionaryM dealloc] + 128
	6   libobjc.A.dylib                     0x000000010a689604 objc_object::sidetable_release(bool, bool) + 174
	7   libobjc.A.dylib                     0x000000010a6858ad _object_remove_assocations + 562
	8   libobjc.A.dylib                     0x000000010a682c02 objc_destructInstance + 84
	9   libobjc.A.dylib                     0x000000010a688d88 -[NSObject dealloc] + 21
	10  FBSDKCoreKit                        0x00000001074e3a40 -[FBSDKGraphReque<…>
2022-01-18 09:29:44.217 I  Smokeless Stage[80550:6ab3ca] [com.wix.Detox:DetoxInstrumentsManager] Called stop but no recording in progress
2022-01-18 09:29:44.218 I  Smokeless Stage[80550:6ab3c0] [com.facebook.react.log:javascript] Running "Smokeless" with {"rootTag":1,"initialProps":{}}
2022-01-18 09:29:44.218 I  Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_resolver_create_dns_service_locked [C7] Starting host resolution graph.facebook.com:443, flags 0xc000d000 proto 0
2022-01-18 09:29:44.218 Df Smokeless Stage[80550:6ab4c2] (CFNetwork) Task <FFF4EA71-83E4-4C40-B62E-F8B8513E19EB>.<1> setting up Connection 7
2022-01-18 09:29:44.218 Db Smokeless Stage[80550:6ab4c2] [com.apple.network:activity] Returning should log: 0 for activity cfnetwork:foreground_task
2022-01-18 09:29:44.219 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key UseNewAXBundleLoader in CFPrefsSearchListSource<0x600000cc8e00> (Domain: com.apple.Accessibility, Container: /Users/smokeless/Library/Developer/CoreSimulator/Devices/231BD033-82F0-4945-AABC-A1C74857A7A2/data)
2022-01-18 09:29:44.219 Db Smokeless Stage[80550:6aaec6] [com.apple.defaults:User Defaults] found no value for key UseNewAXBundleLoader in CFPrefsSearchListSource<0x600000cc0980> (Domain: com.apple.Accessibility, Container: (null))
2022-01-18 09:29:44.219 I  Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_connection_create_with_id [C8] create connection to Hostname#1b540f1a:443
2022-01-18 09:29:44.219 Df Smokeless Stage[80550:6ab4c2] (CFNetwork) Connection 8: enabling TLS
2022-01-18 09:29:44.219 Df Smokeless Stage[80550:6ab4c2] (CFNetwork) Connection 8: starting, TC(0x0)
2022-01-18 09:29:44.219 Df Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] [C8 AD919394-6DC3-41ED-9364-EE07D5296C5E graph.facebook.com:443 tcp, url hash: 3461e81d, tls, context: com.apple.CFNetwork.NSURLSession.{D0C92558-A5CA-4727-B345-410D25D7FD2F}{(null)}{Y}{2}, proc: D6C13870-EF95-30D6-82EC-D09C6A8801F9] start
2022-01-18 09:29:44.219 I  Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_endpoint_handler_start [C8 graph.facebook.com:443 initial path ((null))]
2022-01-18 09:29:44.219 Df Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] [C8 graph.facebook.com:443 initial path ((null))] event: path:start @0.000s
2022-01-18 09:29:44.220 Db Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_endpoint_handler_path_change [C8 graph.facebook.com:443 waiting path (satisfied (Path is satisfied), interface: en0)]
2022-01-18 09:29:44.220 Df Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] [C8 graph.facebook.com:443 waiting path (satisfied (Path is satisfied), interface: en0)] event: path:satisfied @0.000s, uuid: 1ABB9475-19A4-417A-A9AC-8CC8444CB796
2022-01-18 09:29:44.220 I  Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_connection_endpoint_report_on_nw_queue [C8 graph.facebook.com:443 waiting path (satisfied (Path is satisfied), interface: en0)] skipping state update
2022-01-18 09:29:44.220 Db Smokeless Stage[80550:6ab4c2] [com.apple.network:] nw_endpoint_proxy_handler_should_use_proxy Looking up proxy for hostname: graph.facebook.com, ifindex: 0
2022-01-18 09:29:44.220 Db Smokeless Stage[80550:6ab3c9] [com.apple.defaults:User Defaults] found no value for key UseNewAXBundleLoader in CFPrefsSearchListSource<0x600000cc8e00> (Domain: com.apple.Accessibility, Container: /Users/smokeless/Library/Developer/CoreSimulator/Devices/231BD033-82F0-4945-AABC-A1C74857A7A2/data)
2022-01-18 09:29:44.220 Db Smokeless Stage[80550:6ab4c2] [com.apple.SystemConfiguration:SCPreferences] create <SCPreferences 0x7fad15d05fd0 [0x10adb5b90]> {name = SCDynamicStoreCopyProxiesWithOptions, id = /Library/Managed Preferences/mobile/com.apple.SystemConfiguration.plist, path = /Library/Managed Preferences/mobile/com.apple.SystemConfiguration.plist}
2022-01-18 09:29:44.221 Db Smokeless Stage[80550:6ab3c9] [com.apple.defaults:User Defaults] found no value for key UseNewAXBundleLoader in CFPrefsSearchListSource<0x600000cc0980> (Domain: com.apple.Accessibility, Container: (null))
2022-01-18 09:29:44.221 Db Smokeless Stage[80550:6ab4c2] [com.apple.SystemConfiguration:SCPreferences] SCPreferences() access: /Library/Managed Preferences/mobile/com.apple.SystemConfiguration.plist, size=0
2022-01-18 09:29:44.221 Db Smokeless Stage[80550:6ab4c2] [com.apple.SystemConfiguration:SCPreferences] release <SCPreferences 0x7fad15d05fd0 [0x10adb5b90]> {name = SCDynamicStoreCopyProxiesWithOptions, id = /Library/Managed Preferences/mobile/com.apple.SystemConfiguration.plist, path = /Library/Managed Preferences/mobile/com.apple.SystemConfiguration.plist, accessed}
2022-01-18 09:29:44.222 Db Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] -[NWConcrete_nw_endpoint_resolver startWithHandler:] [C8 graph.facebook.com:443 waiting resolver (satisfied (Path is satisfied), interface: en0)]
2022-01-18 09:29:44.222 Df Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] [C8 graph.facebook.com:443 in_progress resolver (satisfied (Path is satisfied), interface: en0)] event: resolver:start_dns @0.002s
2022-01-18 09:29:44.222 Df Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_connection_report_state_with_handler_on_nw_queue [C8] reporting state preparing
2022-01-18 09:29:44.223 I  Smokeless Stage[80550:6ab4c2] [com.apple.network:connection] nw_resolver_create_dns_service_locked [C8] Starting host resolution graph.facebook.com:443, flags 0xc000d000 proto 0
2022-01-18 09:29:44.223 Df Smokeless Stage[80550:6ab4c2] (CFNetwork) Task <2F491C89-9CBE-4817-94BD-0E35357230F3>.<1> setting up Connection 8
2022-01-18 09:29:44.223 Db Smokeless Stage[80550:6ab4c2] [com.apple.network:activity] Returning should log: 0 for activity cfnetwork:foreground_task
 </code>
</details>

**Comment by asafkorem** [comment]
So the answer for my question [here](https://github.com/wix/Detox/issues/3168#issuecomment-1015175992) is not?
> are you using Firebase-Performance in your project?

**Comment by sandeep-bastola** [comment]
Updating the Firebase-Performance to latest V14.2.2 resolves the crash issue, but app gets stuck many times and timeOut happens.

**Comment by asafkorem** [comment]
Only on iOS and when running Detox?

**Comment by asafkorem** [comment]
@sandeep-bastola we need more information regarding the the new issue. 
Please follow the instructions under [Identifying which synchronization mechanism causes us to wait too much](https://wix.github.io/Detox/docs/troubleshooting/synchronization#identifying-which-synchronization-mechanism-causes-us-to-wait-too-much) & [synchronization debug documentation](https://github.com/wix/DetoxSync/blob/master/IdleStatusDocumentation.md) and provide us with the relevant synchronization debug output.

**Comment by dbilgin** [comment]
I had the signal 11 issue, after updating the firebase dependencies that got fixed, I just had the occasional problem with the Network Request event waiting for codepush, but that goes away eventually.

**Comment by dbilgin** [comment]
Just a small update from my side, I am running these tests on bitrise and I saw that on v19.4.2 the tests are failing too frequently, so I checked it out and it showed that it's getting stuck often with the message below for a very long time:
```
detox[16771] INFO:  [APP_STATUS] The app is busy with the following tasks:
• There are 1 work items pending on the dispatch queue: "Main Queue (<OS_dispatch_queue_main: com.apple.main-thread>)".
• Run loop "Main Run Loop" is awake.
• UI elements are busy:
  - Layer animations pending: 1.
```

I am sure there are no long lasting animations in my app so I decided to make video recordings of the tests as artifacts.
It goes a bit weird, one test fails and then all the remaining tests also fail, the first couple of tests had no video at all and the ones coming after had signal 11s in the video. I didn't encounter this locally though.

**Comment by asafkorem** [comment]
@dbilgin I see that you opened another issue for your problem: https://github.com/wix/Detox/issues/3185, let's continue there.

> Updating the Firebase-Performance to latest V14.2.2 resolves the crash issue, but app gets stuck many times and timeOut happens.

@sandeep-bastola please open another issue for that crash (sounds unrelated to this issue, which was solved after firebase update) and provide the logs as requested here: https://github.com/wix/Detox/issues/3176#issuecomment-1015539419

I'm closing this one.
